src/univariate/continuous/beta.jl

```diff
@@ -32,19 +32,15 @@ struct Beta{T<:Real} <: ContinuousUnivariateDistribution
     Beta{T}(α::T, β::T) where {T} = new{T}(α, β)
 end
 
-function Beta(α::T, β::T) where {T<:Real}
-    @check_args(Beta, α > zero(α) && β > zero(β))
-    return Beta{T}(α, β)
-end
-
-function Beta(α::T, β::T, ::NoArgCheck) where {T<:Real}
+function Beta(α::T, β::T; check_arg=true) where {T<:Real}
+    check_arg && @check_args(Beta, α > zero(α) && β > zero(β))
     return Beta{T}(α, β)
 end
 
 Beta(α::Real, β::Real) = Beta(promote(α, β)...)
 Beta(α::Integer, β::Integer) = Beta(float(α), float(β))
 Beta(α::Real) = Beta(α, α)
-Beta() = Beta(1.0, 1.0, NoArgCheck())
+Beta() = Beta(1.0, 1.0, check_arg = false)
```

Comment:
Not sure what's the most common style in this repo, but having spaces around `=` for keyword arguments here is inconsistent with above.